Sources/Forum.php

```diff
@@ -143,26 +160,12 @@ class Forum
 	 *    log visits to index.php?action=pm;sa=popup, but other sub-actions
 	 *    like index.php?action=pm;sa=send will be logged.
 	 */
-	public static $unlogged_actions = [
+	public static array $unlogged_actions = [
 		'about:unknown' => true,
+		'calendar' => ['sa' => ['clock']],
 		'clock' => true,
-		'dlattach' => true,
-		'findmember' => true,
-		'helpadmin' => true,
 		'jsoption' => true,
-		'likes' => true,
 		'modifycat' => true,
```

Comment:
These actions cannot use the new approach because they are simple callbacks that piggyback off other actions. any reason that they do not use their own objects? The only reason I can think of is that they are easter eggs and therefore require hunting around.